pyanaconda/errors.py

```diff
@@ -148,7 +148,7 @@ def _storageResetHandler(self, exn):
                      "can access by pressing ctrl-alt-f1 and then ctrl-b 2."
                      "\n\nOnce you have resolved the issue you can retry the "
                      "storage scan. If you do not fix it you will have to exit "
-                     "the installer.") % {"errortxt": escape_markup(exn.message)})
+                     "the installer.") % {"errortxt": escape_markup(str(exn))})
```

Comment:
I think the ``str()`` call is not necessary in this case, but it can hardly hurt so feel free to keep it in place.